java/arcs/core/data/testdata/ConsumeProduce.arcs

```diff
@@ -0,0 +1,23 @@
+meta
+  namespace: arcs.core.data.testdata.gen
+
+import './Schemas.arcs'
+import './Produce.arcs'
```

Comment:
why do you need to import Produce.arcs?

Reply:
recipe2plan operates in terms of a given manifest, if you don't import the .arcs file from which you map it will not be able to find a store.

... but. What we could do is to refactor things so that you can pass multiple files to recipe2plan and then construct a manifest that imports all the passed files and that would be your context?

Reply:
oh, i see. 
another option is to create a single top level arcs file that imports everything, like the "canonical" we used to have in the web arcs.